app/Policies/SettingsPolicy.php

```diff
@@ -68,6 +69,19 @@ public function canSeeDiagnostics(User $user): bool
 		return false;
 	}
 
+	/**
+	 * This function returns false as it is bypassed by the before()
+	 * which directly checks for admin rights.
+	 *
+	 * @param User $user
+	 *
+	 * @return bool
+	 */
+	public function canSeeStatistics(User $user): bool
+	{
+		return true;
```

Comment:
No, we allow users to access statistics, the difference is that it will be filtered.
Note that not logged in user will not be able to access those data because the parameter of the function is `User $user` instead of `?User $user` (nullable hence guest).